crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -69,6 +69,20 @@ impl<'w> EntityRef<'w> {
         }
     }
 
+    #[inline]
+    pub fn get_change_ticks<T: Component>(&self) -> Option<&'w ComponentTicks> {
+        // SAFE: entity location is valid and returned component is of type T
+        unsafe {
+            get_component_and_ticks_with_type(
```

Comment:
I know you're trying to reuse code here, but we don't need to retrieve the component for this operation. This seems like the sort of small inefficiency that we'd merge and then forget about. A theoretical `get_ticks` and the existing `get_component` and `get_component_with_ticks` methods probably have some refactoring potential to share code. They all need to "get the entity's offset in each storage". This seems like a relatively trivial refactor for Table storage, but Sparse Set storage would probably need some new get_with_dense_index_unchecked() apis.